__tests__/api/__utils__/seeds/users_and_follow_seeds.js

```diff
@@ -30,7 +52,7 @@ const generateUserAndFollowSeeds = async () => {
     is_backdoor_user: false
   });
 
-  for (let i = 1; i < 70; i++) {
+  for (let i = 2; i < 71; i++) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The loop index has been shifted by one. This change seems to be made to accommodate the anonymous user added in the previous hunk. However, it's important to ensure that this shift doesn't affect other parts of the code where this loop is used.
```diff
-  for (let i = 2; i < 71; i++) {
+  for (let i = 1; i < 70; i++) {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->